src/plugins/notebook/components/Notebook.vue

```diff
@@ -668,26 +657,24 @@ export default {
 
             const notebookStorage = getDefaultNotebook();
             if (!notebookStorage
-                    || notebookStorage.notebookMeta.identifier.key !== this.domainObject.identifier.key) {
+                    || notebookStorage.identifier.key !== this.domainObject.identifier.key) {
                 return;
             }
 
-            const defaultNotebookSection = notebookStorage.section;
+            const defaultNotebookSectionId = notebookStorage.defaultSectionId;
             const section = sections.find(s => s.id === id);
-            if (!section && defaultNotebookSection.id === id) {
-                this.defaultSectionId = null;
-                this.defaultPageId = null;
+            if (!section && defaultNotebookSectionId === id) {
```

Comment:
This can be simplified by having an outer `if(defaultNotebookSectionId === id) {... }` and then putting the logic to check the section and update the section in that if block.